vue-app/babel.config.js

```diff
@@ -1,3 +1,3 @@
-module.exports = {
-  presets: ['@vue/cli-plugin-babel/preset'],
-};
+// module.exports = {
```

Comment:
if app seems to work, then I recommend we remove this file

Reply:
I removed the file. but do we need to remove anything from the `package.json` too?